_posts/archived/2009-12-07-localizing-aspnetmvc-validation.aspx.markdown

```diff
@@ -4,7 +4,8 @@ title: "Localizing ASP.NET MVC Validation"
 date: 2009-12-07 -0800
 comments: true
 disqus_identifier: 18664
-categories: [aspnetmvc]
+redirect_from: "/archive/2009/12/12/localizing-aspnetmvc-validation.aspx"
```

Comment:
Another option could be `/archive/2009/12/12/localizing-aspnetmvc-validation.aspx.html` since Pages will let you drop the `.html` off the end of the url